diff --git a/dist/cli/build/bundle-server.js b/dist/cli/build/bundle-server.js
index be3f893561dab6ad635e71a9cbb49603bfa44534..65567efd5add66fef6c20bfce7febaa8f07afe69 100644
--- a/dist/cli/build/bundle-server.js
+++ b/dist/cli/build/bundle-server.js
@@ -100,7 +100,7 @@ export async function bundleServer(buildOpts, projectOpts) {
             // Apply updater updates, must be the last plugin
             updater.plugin,
         ],
-        external: ["./middleware/handler.mjs"],
+        external: ["./middleware/handler.mjs", "cloudflare:sockets"],
         alias: {
             // When @vercel/og is not used, alias the edge entry to a throwing shim so the
             // dynamic `import("next/dist/compiled/@vercel/og/index.edge.js")` call site
